makelib/common.mk

```diff
@@ -340,6 +340,10 @@ test:
 	@$(MAKE) test.run
 	@$(MAKE) test.done
 
+# integration tests
+test-integration:
```

Comment:
I would have assumed that something so very Crossplane specific would not live in this common build project.

But after taking another look at https://github.com/crossplaneio/crossplane/blob/master/Makefile, it does appear that the kitchen sink is inherited from this file.

Inclusion here may benefit the Crossplane project as each provider is broken out into its own repository. :+1: